src/vm/codeman.cpp

```diff
@@ -1016,6 +1016,35 @@ PTR_VOID GetUnwindDataBlob(TADDR moduleBase, PTR_RUNTIME_FUNCTION pRuntimeFuncti
 
     *pSize = size;
     return xdata;
+
+#elif defined(_TARGET_ARM64_)
+    // compute the size of the unwind info
+    PTR_TADDR xdata = dac_cast<PTR_TADDR>(pRuntimeFunction->UnwindData + moduleBase);
```

Comment:
you should read 4 bytes at a time. This would read 8 byes as PTR_TADDR is 8 bytes on ARM64. So make this DWORD*